<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>837</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1023</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Настройки графика смен</string>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>1. Выберите первый рабочий день</string>
       </property>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>ddd dd.MM.yyyy</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>231</width>
         <height>231</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>2. Установите график работы</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>27</y>
          <width>211</width>
          <height>197</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Название </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Цвет</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_24">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_26">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="comboBox_23">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_25">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="comboBox_11">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_14">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QComboBox" name="comboBox_12">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_16">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="comboBox_13">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBox_17">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QComboBox" name="comboBox_15">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox_18">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="comboBox_21">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дневная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ночная смена</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Отсыпной</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Выходной</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBox_19">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Красный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Оранжевый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Желтый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Зеленый</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Синий</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Рассчитать график</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Добавить событие</string>
      </property>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>1. Выберите день</string>
       </property>
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>ddd dd.MM.yyyy</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>231</width>
         <height>131</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>2. Введите текст заметки</string>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>211</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>231</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>3. Введите адрес эл.почты</string>
       </property>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Отправить напоминание</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>311</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>310</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Здесь будет отображаться информация о выбранной дате - какая смена, содержание заметки при наличии </string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
